docs/docs/shapes.mdx

```diff
@@ -11,15 +11,95 @@ keywords:
   - utils
 ---
 
-In tldraw, **shapes** are the things that are on the canvas. This article is about shapes: what they are, how they work, and how to create your own shapes. If you'd prefer to see an example, see the tldraw repository's [examples app](https://github.com/tldraw/tldraw/tree/main/apps/examples) for examples of how to create custom shapes in tldraw.
+##### Table of Contents
+
+- [Introduction](#introduction)
+- [Types of shape](#types-of-shape)
+- [Inside a shape](#inside-a-shape)
+- [Shape util](#shape-util)
+- [Custom shapes](#custom-shapes)
+
+## Introduction
+
+In tldraw, a **shape** is something that can exist on the page, like an arrow,
+an image, or some text.
+
+This article is about shapes: what they are, how they work, and how to create
+your own shapes. If you'd prefer to see an example, see the tldraw repository's
+[examples app](https://github.com/tldraw/tldraw/tree/main/apps/examples) for
+examples of how to create custom shapes in tldraw.
+
+## Types of shape
+
+The editor comes with some built-in [**core shapes**](/gen/editor/coreShapes),
+like the [text shape](/gen/tlschema/TLTextShape), the [group
+shape](/gen/tlschema/TLGroupShape) and more. These are required for the editor
+to work, so they can't be removed.
+
+There are also some [**default shapes**](/gen/editor/defaultShapes) available,
+like the [arrow shape](/gen/tlschema/TLArrowShape), the [draw
+shape](/gen/tlschema/TLDrawShape), and more. The
+[`<Tldraw>`](/gen/editor/Editor) component provides access to these shapes
+automatically.
```

Comment:
Here, or somewhere else (perhaps the Usage page), we could outline how to *not* include some of these shapes by using the `<TldrawEditor>` component.